### PR TITLE
INNO-254: change age squares from image to css

### PR DESCRIPTION
**Pull request by lordmcfuzz**

### CODE REVIEW

File: innovation.game.php [hunk 1]
```diff
@@ -2583,7 +2586,7 @@ function sendNotificationWithOnePlayerInvolved($message_for_player, $message_for
             $notif_args_for_others['name'] = self::getCardName($card['id']);
             // TODO(LATER): We should stop sending the card's properties which aren't actually used.
             $notif_args_for_others = array_merge($notif_args_for_others, $card);
-        } else if (array_key_exists('<<<', $delimiters_for_player)) {
+        } else if (array_key_exists('<<<', $delimiters_for_others)) {
```
Comment (lordmcfuzz) [inline innovation.game.php:2589]: I noticed this... not sure if it should actually be changed.

File: innovation.game.php [hunk 2]
```diff
@@ -2602,6 +2605,10 @@ function sendNotificationWithOnePlayerInvolved($message_for_player, $message_for
             $notif_args_for_others['type'] = $card['type'];
             $notif_args_for_others['is_relic'] = $card['is_relic'];
         }
+        if (array_key_exists('>', $delimiters_for_others)) {
+            // replace ${age} with card age
+            $notif_args_for_others['>'] =str_replace("\${age}", $card['age'],$notif_args_for_others['>']);
+        }
         
         self::notifyPlayer($player_id, "transferedCard", $message_for_player, $notif_args_for_player);
         self::notifyAllPlayersBut($player_id, "transferedCard", $message_for_others, $notif_args_for_others);
```
Comment (lordmcfuzz) [inline innovation.game.php:2614]: Oops totally missed testing /fixing Two players in the lower code.

File: innovation.scss [hunk 3]
```diff
@@ -1144,36 +1110,14 @@
...
-        }
+        &.age {
+			@include square-age;
+			font-size: inherit;
+			line-height: 19px;
+			&.age_10 {
+				@include square-age-10;
+			}
+		}
     }
```
Comment (lordmcfuzz) [inline innovation.scss:1121]: I need to add an in_tooltip age section as there is an 8 box in a special achievement tool tip

**Review by micahstairs — APPROVED** [review]
Amazing! I think we might tweak some of the sizing a bit, but we can do that later.

Also, I made a simplification to remove str_replace.

**Pull request merged**